### PR TITLE
Updated links

### PR DESCRIPTION
**Pull request by mbrickn**
Howdy!

I just updated some links to use https instead of http.

Thank you! ^ _ ^

### CODE REVIEW

**Comment by bep** [comment]
Have you checked that the sites in question actually works over https?

File: README.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 ![Hugo](https://raw.githubusercontent.com/gohugoio/hugoDocs/master/static/img/hugo-logo.png)
 
-A Fast and Flexible Static Site Generator built with love by [spf13](http://spf13.com/) and [friends](https://github.com/gohugoio/hugo/graphs/contributors) in [Go][].
+A Fast and Flexible Static Site Generator built with love by [spf13](https://spf13.com/) and [friends](https://github.com/gohugoio/hugo/graphs/contributors) in [Go][].
```
Comment (DirtyF) [inline README.md]: FWIW @spf13 website does not display styles when served over https here.

Comment (mbrickn) [inline README.md]: My bad. I thought the SPF13 website was really minimalist. I wouldn't have made this request if it hadn't loaded at all or anything like that.

Comment (mbrickn) [inline README.md]: Sorry for wasting your time. > _ <

Comment (mbrickn) [inline README.md]: Again 100% my fault, really sorry about this!

**Pull request closed without merging**